package.json

```diff
@@ -3,20 +3,79 @@
   "version": "1.0.0",
   "description": "",
   "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
+  "license": "MIT",
   "devDependencies": {
-    "@nomiclabs/hardhat-ethers": "^2.0.1",
+    "@commitlint/cli": "^9.1.2",
+    "@commitlint/config-conventional": "^9.1.2",
+    "@ethersproject/abstract-signer": "^5.0.6",
+    "@ethersproject/bignumber": "^5.0.8",
+    "@nomiclabs/hardhat-ethers": "^2.0.0",
     "@nomiclabs/hardhat-etherscan": "^2.1.0",
-    "@nomiclabs/hardhat-waffle": "^2.0.1",
-    "@openzeppelin/hardhat-upgrades": "^1.6.0",
-    "@openzeppelin/test-helpers": "^0.5.10",
+    "@nomiclabs/hardhat-ganache": "^2.0.0",
+    "@nomiclabs/hardhat-waffle": "^2.0.0",
+    "@typechain/ethers-v5": "^4.0.0",
+    "@types/chai": "^4.2.13",
+    "@types/fs-extra": "^9.0.1",
+    "@types/mocha": "^7.0.2",
+    "@types/node": "^14.11.8",
+    "@typescript-eslint/eslint-plugin": "^3.10.1",
+    "@typescript-eslint/parser": "^3.10.1",
     "chai": "^4.2.0",
+    "commitizen": "^4.2.1",
+    "cz-conventional-changelog": "^3.3.0",
     "dotenv": "^8.2.0",
-    "ethereum-waffle": "^3.2.1",
-    "ethers": "^5.0.26",
-    "hardhat": "^2.0.8",
-    "hardhat-contract-sizer": "^2.0.2"
+    "eslint": "^7.11.0",
+    "eslint-config-prettier": "^6.12.0",
+    "ethereum-waffle": "^3.2.0",
+    "ethers": "^5.0.17",
+    "fs-extra": "^9.0.1",
+    "hardhat": "^2.0.4",
+    "hardhat-contract-sizer": "^2.0.2",
+    "hardhat-typechain": "^0.3.3",
+    "husky": "^4.3.0",
+    "mocha": "^8.2.1",
+    "prettier": "^2.1.2",
+    "prettier-plugin-solidity": "^1.0.0-beta.1",
+    "shelljs": "^0.8.4",
+    "solc": "0.7.4",
+    "solhint": "^3.2.1",
+    "solhint-plugin-prettier": "^0.0.5",
+    "solidity-coverage": "^0.7.12",
+    "ts-generator": "^0.1.1",
+    "ts-node": "^8.10.2",
+    "typechain": "^3.0.0",
+    "typescript": "^3.9.7"
+  },
+  "dependencies": {
+    "@openzeppelin/contracts": "^4.0.0-rc.0"
```

Comment:
I'd prefer if we use an official stable build instead of a rc branch if there's no specific need for it.

Reply:
There's no official build, this is for test contracts only

Reply:
Stable 4.0.0 has been released